standalone/loadListInGlobalVariable.js

```diff
@@ -0,0 +1,9 @@
+//@zaproxy-standalone
+
+org.zaproxy.zap.extension.script.ScriptVars.setGlobalVar('LIST', JSON.stringify(['Zaproxy', 'Zap', 'Simon', 'Mozilla']))
+
+list = JSON.parse(org.zaproxy.zap.extension.script.ScriptVars.getGlobalVars()['LIST'])
```

Comment:
Might worth adding a comment saying that this will work only with ZAP < 2.7.0 (or use `getGlobalVar('LIST')`?).

Reply:
Yes sure, I didn't know about it